common/common.scss

```diff
@@ -265,3 +265,13 @@ aside.sidebar.full.tablet {
 .show-more {
   text-align: center;
 }
+
+.layouts-category-list.children-default-expanded ul.parent-categories {
+  li.layouts-category-link.showing-children,
+  .layouts-category-list
+    ul.parent-categories
+    li.layouts-category-link:hover:not(.no-link),
+  ul.child-categories {
+    background-color: transparent;
```

Comment:
What's the reason for this CSS change? Doesn't the existing CSS support showing child categories?

Reply:
Normally the when subcategories are collapsed by default the CSS looks good. However, when everything is expanded at once, I thought the highlighting might be a bit too much. Which is why I added that change. But I can remove it as its not crucial.
![Uploading Screen Shot 2022-01-06 at 5.06.27 PM.png…]()